src/registrar/assets/js/get-gov.js

```diff
@@ -1170,15 +1231,15 @@ document.addEventListener('DOMContentLoaded', function() {
 
           // Do not scroll on first page load
           if (loaded)
-            ScrollToElement('id', 'domains-header');
+            ScrollToElement('class', 'domains');
           hasLoaded = true;
 
           // update pagination
           updatePagination(
             'domain',
             '#domains-pagination',
             '#domains-pagination .usa-pagination__counter',
-            '#domains-header',
+            '#domains',
```

Comment:
Yeah, the IDs are a bit flimsy because the headers can be swapped out for different users, so let's scroll to the immutable class and set the pagination href value to the id